### ISSUE TITLE
No directory tree on Manage

### ISSUE THREAD

**Issue by titomontes** [CouchPotato/CouchPotatoServer#7049]
### Steps to reproduce:
1. Fresh install from source
2. Whenever I try to set my main movies folder under Manage, once I click on the dialog box it returns an empty box with Cancel and Clear/Save buttons only. Address can be entered manually, but it gets changed to duplicate backslashes (from \ to \\) on every restart, so its rendered useless.

I´ve tried this several times erasing DB and even on a virgin PC

### Information:
Version
4a0999d9 (‎06‎/‎09‎/‎2016‎ ‎11‎:‎53‎:‎22)
Updater
git, master
ID
1776 
Directories
C:\Program Files\CouchPotato
C:\WINDOWS\system32\config\systemprofile\AppData\Roaming\CouchPotato
Startup Args
[]
Namespace(config_file='C:\\WINDOWS\\system32\\config\\systemprofile\\AppData\\Roaming\\CouchPotato\\settings.conf', console_log=False, daemon=False, data_dir=None, debug=False, pid_file='C:\\WINDOWS\\system32\\config\\systemprofile\\AppData\\Roaming\\CouchPotato\\couchpotato.pid', quiet=False)

### Logs:
No log

**Comment by Thijxx** [comment]
![selection_411](https://cloud.githubusercontent.com/assets/3355823/19736564/184c6b38-9bb0-11e6-9af6-4d9e8671fc1f.png)

for me it's impossible to navigate the tree, same problem on phone or tablet

**Comment by destrugter** [comment]
I am having the same issue as these guys. Instead of reporting a new bug I'll just post here. I think Thijxx is correct, the list is there it's just cut off!

**Comment by clinton-hall** [comment]
Have you tried another browser?

**Comment by Thijxx** [comment]
Yes. Firefox, chrome, safari and all of those on iOS and Linux. Bug is persistent.

**Comment by clinton-hall** [comment]
what happens if you save one of the visible folders (logs) then stop CP and manually edit the C:\WINDOWS\system32\config\systemprofile\AppData\Roaming\CouchPotato\settings.conf
and replace the logs file location with the wanted manage folder??

note, in the conf you will need ``\\` in place of `\`
i.e. `C;\\user\\<name>\\documents\\movies`

**Comment by titomontes** [comment]
I tried that, but did not know that two backslashes were needed. I'll try and tell asap. Problem is still there,  but kind of circumvented it by blindly typing the route when the tree menu is displayed. Sometimes it sticks, others it doesn't, don't know why. Also, when it doesn't, the two backslashes appear on the route box (in manage a well as in renamer), and access to the directory is lost, so any rescan fails. 
Hope this helps

**Comment by Thijxx** [comment]
Yes editing the config file works fine, not with the Windows paths but with normal sensible Linux paths :)

**Comment by titomontes** [comment]
OK, just did that; It worked fine on Renamer (it shows the correct path), but Manage keeps showing two backslashes, rendering the location useless. Also, renamer doesn´t work anymore.

**Comment by titomontes** [comment]
I didn´t notice that you asked to save one of the visible folders. Well, there are none :(

**Comment by Thijxx** [comment]
Jus tested it on a clean install with a clean clone from Git and same problem. Cannot set directories, it also crashes silently, systemd shows OK while Couch died (over missing directories probably)
